src/runtime/handle.ts

```diff
@@ -96,12 +102,12 @@ export abstract class Handle {
     }
   }
 
-  _serialize(entity) {
+  _serialize(entity: EntityInterface) {
     assert(entity, 'can\'t serialize a null entity');
     if (!entity.isIdentified()) {
       entity.createIdentity(this._proxy.generateIDComponents());
     }
-    const id = entity[Symbols.identifier];
+    const id = entity.id;
```

Comment:
there was a notion at one stage that the id would be at entity[Symbols.identifier] so you could only extract it if you had access to Symbols.identifier. Not sure if we want to keep it or not; but switching to entity.id is a behaviour change.

Reply:
Oh is that how that works? Neat.

I was just experimenting with this, didn't mean to actually include this change. Reverted.

Reply:
Nope, I did it for a reason:

Type 'symbol' cannot be used as an index type.

See https://github.com/Microsoft/TypeScript/issues/1863

I've cast the symbol to any to get past this check. Any better ideas?